_drafts/2019-10-01-hacktoberfest-is-here.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: "Hacktoberfest 2019 is here"
+author: [herflis]
+image: "../img/posts/hacktoberfest2019.png"
+tags: [open source, hacktoberfest, community]
+
+---
+
+<p align="center">
+  <img src="../img/posts/hacktoberfest2019.png">
+</p>
+
+# Hacktoberfest 2019 is here
+
+Open source and community were key for us since the beginning of product development at Sense/Net. Our company is very committed to supporting the movement with helping developers expanding their skill set and make the community bigger and bigger not just around sensenet but globally.
```

Comment:
```suggestion
Open source and community were key for us since the beginning of product development at Sense/Net. Our company is very committed to support the movement with helping developers expanding their skill set and make the community bigger and bigger not just around sensenet but globally.
```